README.md

```diff
@@ -0,0 +1,69 @@
+[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+
+## Introduction
+
+> *App introduction goes here ...*
```

Comment:
How about filling in the info for this project already?

Reply:
Added a description for the project in 29970bd.